src/components/Blog.jsx

```diff
@@ -14,22 +14,30 @@ function ShowPaper({ paper }) {
   }
 
   return (
-    <div className="shadow-secondary/5 mb-6 space-y-8 rounded-2xl bg-black/60 py-14 px-10 text-start shadow-xl backdrop-blur">
-      <p className="text-xl font-semibold tracking-tight text-jordi-orange-600">
+    <div className="mb-6 space-y-8 rounded-2xl bg-jordi-beige/60 py-14 px-10 text-start shadow-xl shadow-black/5 backdrop-blur dark:bg-black/60">
+      <p className="text-xl font-semibold tracking-tight text-jordi-orange-500">
         {paper.author}
       </p>
-      <p className="mt-1 font-mono text-sm text-jordi-orange-700">
+      <p className="mt-1 font-mono text-sm text-jordi-black dark:text-jordi-beige-500">
         {paper.course}
       </p>
-      <p className="mt-1 tracking-tight text-jordi-orange-600">{paper.title}</p>
-      <p className={`text-base ${open ? 'text-white' : 'hidden'}`}>
+      <p className="mt-1 tracking-tight text-jordi-orange-400">{paper.title}</p>
+      <p
+        className={`text-base ${
+          open ? 'text-jordi-black dark:text-jordi-beige' : 'hidden'
+        }`}
+      >
         {paper.text}
       </p>
-      <p className={`text-base ${open ? 'hidden' : 'text-white/70'}`}>
+      <p
+        className={`text-base ${
+          open ? 'hidden' : 'text-jordi-black/70 dark:text-jordi-beige/70'
+        }`}
+      >
         {paper.summary}
       </p>
       <Button
-        className="mx-[40%] bg-jordi-orange-700 text-white"
+        className="mx-[40%] bg-jordi-orange-500 text-white"
```

Comment:
```suggestion
        className="mx-auto bg-jordi-orange-500 text-white"
```
I think the button looks better like this :)